api/reference/metric_rollups.md

```diff
@@ -24,20 +24,23 @@ Supported query parameters:
 
 ``` data
 GET /api/metric_rollups?resource_type=VmOrTemplate
-                      &resource_ids=[101,102,103]
-                      &capture_interval=daily
-                      &start_date='2017-10-01'
-                      &limit=60
+                       &resource_ids[]=101
```

Comment:
Should we also update the table above on line 18 ?

```
| resource\_ids     | \[id1,id2,id3, …​\]
```

```
| resource\_ids[]     | id1
| resource\_ids[]     | id2
```

Reply:
yes, I updated it as you suggested (+ `resource\_ids[]     | ...  `)

thanks @abellotti